### PR TITLE
chore(test): move mockmodule_spec.js off of the control flow

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

**Comment by cnishina** [comment]
Reference #4995

File: spec/basic/mockmodule_spec.js [hunk 1]
```diff
@@ -1,99 +1,102 @@
...
   // This module shows the use of a string for the load
   // function.
-  var mockModuleB = `angular.module('moduleB', []).value('version', '3');`;
+  let mockModuleB = `angular.module('moduleB', []).value('version', '3');`;
 
   // A third module overriding the 'version' service. This function
   // references the additional arguments provided through addMockModule().
-  var mockModuleC = function() {
+  let mockModuleC = () => {
     var newModule = angular.module('moduleC', []);
```
Comment (CrispusDH) [inline spec/basic/mockmodule_spec.js]: should it be `let`?

File: spec/basic/mockmodule_spec.js [hunk 2]
```diff
@@ -1,99 +1,102 @@
-describe('mock modules', function() {
+describe('mock modules', () => {
   // A module to override the 'version' service. This function will be
   // executed in the context of the application under test, so it may
   // not refer to any local variables.
-  var mockModuleA = function() {
-    var newModule = angular.module('moduleA', []);
+  let mockModuleA = () => {
```
Comment (CrispusDH) [inline spec/basic/mockmodule_spec.js]: this variable could be `const`

File: spec/basic/mockmodule_spec.js [hunk 3]
```diff
@@ -1,99 +1,102 @@
...
+    let newModule = angular.module('moduleA', []);
     newModule.value('version', '2');
   };
 
   // A second module overriding the 'version' service.
   // This module shows the use of a string for the load
   // function.
-  var mockModuleB = `angular.module('moduleB', []).value('version', '3');`;
+  let mockModuleB = `angular.module('moduleB', []).value('version', '3');`;
```
Comment (CrispusDH) [inline spec/basic/mockmodule_spec.js]: this variable could be `const`

File: spec/basic/mockmodule_spec.js [hunk 4]
```diff
@@ -1,99 +1,102 @@
...
 
   // A third module overriding the 'version' service. This function
   // references the additional arguments provided through addMockModule().
-  var mockModuleC = function() {
+  let mockModuleC = () => {
```
Comment (CrispusDH) [inline spec/basic/mockmodule_spec.js]: this variable could be `const`

File: spec/basic/mockmodule_spec.js (hunk 2 above)
Comment (cnishina) [reply spec/basic/mockmodule_spec.js]: done.

File: spec/basic/mockmodule_spec.js (hunk 4 above)
Comment (cnishina) [reply spec/basic/mockmodule_spec.js]: done.

File: spec/basic/mockmodule_spec.js (hunk 3 above)
Comment (cnishina) [reply spec/basic/mockmodule_spec.js]: done.

**Pull request merged**